plugins/remotefortressreader/item_reader.cpp

```diff
@@ -402,7 +395,7 @@ void CopyItem(RemoteFortressReader::Item * NetItem, df::item * DfItem)
             case df::enums::improvement_type::ART_IMAGE:
             {
                 VIRTUAL_CAST_VAR(artImage, df::itemimprovement_art_imagest, improvement);
-                CopyImage(artImage->image, netImp->mutable_image());
+                CopyImage(artImage->getImage(DfItem), netImp->mutable_image());
```

Comment:
Does this mean you don't need the offset because there was a vmethod already that worked, or am I misinterpreting this?

Reply:
The offsets I found were actually just for generating new, empty, art images, and allocating them, so they don't actually work for what I need.

The Vmethod only works for decorations with images on them.

Statues don't have such a handy vmethod (though I did find one that just returns the image reference) 
In DF, the method for statues is hidden deep inside the description generating function, so what I'll just do for that is keep pointer to the art_image-*.dat loading function in df-structures, bonus that it's easy to find, and use that when needed, and just ignore statues where the image isn't already generated yet.